runtime/runtime-common/src/constants.rs

```diff
@@ -17,4 +17,10 @@ pub mod currency {
 	pub const fn deposit(items: u32, bytes: u32) -> Balance {
 		items as Balance * 2_000 * CENTS + (bytes as Balance) * 100 * MILLICENTS
 	}
+
+	/// Constant values used within the runtime.
+	pub const MILLIGGX: Balance = 1_000_000_000_000_000;
```

Comment:
Why do we need two impls for UNITS and for MILLIGGX. Alogngside with two deposit functions?

Reply:
no, i will remove UNITS and merge deposit function